src/constants/constants.js

```diff
@@ -0,0 +1,9 @@
+const HEADER_CELLS = [
+    '',
+    'Name',
+    'Type',
+    'Weight',
+    'Abilities'
+];
+
+export default HEADER_CELLS;
```

Comment:
It is better not to use default export here. Just export each constant you create and import it like this 
```
import { HEADER_CELLS } from '../../constants'
```

Also you can name file just _index.js_ and not to specify it in import. It always goes to index.